test/unit/middleware/setLocals.js

```diff
@@ -15,25 +15,12 @@ describe('setLocals', () => {
       let res;
 
       before('setup the function and execute it', () => {
-        req = {
-          query: {
-            location: 'location',
-          },
-          get: getNull,
-        };
-        res = {
-          locals: {
-            backLink: {},
-          },
-        };
+        req = { query: { }, get: getNull };
+        res = { locals: { backLink: {} } };
```

Comment:
Just a bit of a tidy up, trying to reduce the length of the file as much as anything